webclient/src/views/view/view-view.js

```diff
@@ -519,6 +524,25 @@ navigatum.registerView("view", {
         );
       }
     },
+    loadModalRoomfinderMap: function () {
+      const map = this.view_data.maps.roomfinder.available[this.state.map.roomfinder.selected_index];
+
+      const rect = document
+          .getElementById("roomfinderModal-container")
+          .getBoundingClientRect();
+      // -1023px, -1023px is top left corner, 16px = 2*8px is element padding
+      this.state.map.roomfinder.modalX =
+          -1023 + (map.x / map.width) * (rect.width - 65);
+
+      // We cannot use "height" here as it might be still zero before layouting
+      // finished, so we use the aspect ratio here.
+      this.state.map.roomfinder.modalY =
+          -1023 +
+          (map.y / map.height) * (rect.width - 65) * (map.height / map.width);
+    },
+    delayedLoadModalRoomfinderMap: function () {
+      setTimeout(this.loadModalRoomfinderMap, 1000);
```

Comment:
@octycs as you have worked with the other kinds of timeout more than I have:
Do you think this is a good option, or should we experiment with `$nextTick`

Reply:
We will fix this in post if we notice this being an issue